apps/site/src/lib/components/Button/Button.tsx

```diff
@@ -20,8 +20,6 @@ const Button: React.FC<ButtonProps> = ({
 		return (
 			<Link
 				href={href}
-				target={isLightVersion ? "" : "_blank"}
```

Comment:
I removed this so a new tab doesn't open when clicking button link.

This affects other pages that use the same Button component. However, imo none of them should open new tabs. If we need that functionality for another button, we can implement a condition by modifying the interface.

The affected buttons are below:

Home Page - Apply button
Home Page - Mentor and Volunteer buttons
Application Page - Submission button
Navbar - Login button